.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
The following files must be removed from this PR:

.vs/VSWorkspaceState.json
.vs/docs/FileContentIndex/426e5a15-bb6a-42d5-a75e-f8f1f1c0d89b.vsidx
.vs/docs/FileContentIndex/4344c5be-b400-4bd6-b8a5-e31f8673206a.vsidx
.vs/docs/FileContentIndex/71177040-e0e0-4bdf-b4d1-74cf433ef051.vsidx
.vs/docs/FileContentIndex/ecf4d5c5-c83d-4472-9b5a-7eb80d2ed8a7.vsidx
.vs/docs/FileContentIndex/read.lock
.vs/docs/v17/.wsuo
.vs/docs/v17/workspaceFileList.bin
.vs/slnx.sqlite